include/boost/geometry/algorithms/detail/direction_code.hpp

```diff
@@ -188,27 +205,27 @@ struct direction_code_impl<spherical_equatorial_tag>
 template <>
 struct direction_code_impl<spherical_polar_tag>
 {
-    template <typename Point1, typename Point2>
-    static inline int apply(Point1 segment_a, Point1 segment_b,
+    template <typename PointSegmentA, typename PointSegmentB, typename Point2>
+    static inline int apply(PointSegmentA segment_a, PointSegmentB segment_b,
                             Point2 p)
     {
         typedef math::detail::constants_on_spheroid
```

Comment:
Since you replacing typedefs maybe also replace those?

Reply:
:heavy_check_mark: thanks